src/Symfony/Bridge/Twig/Extension/DumpExtension.php

```diff
@@ -23,10 +24,12 @@
 class DumpExtension extends \Twig_Extension
 {
     private $cloner;
+    private $dumper;
 
-    public function __construct(ClonerInterface $cloner)
+    public function __construct(ClonerInterface $cloner, DataDumperInterface $dumper = null)
```

Comment:
I don't know yet... But I know I need a feature like the one you're working here to finish #17290 :-)